e2e/test-component/scenarios/embedding-sdk/locale-on-provider.cy.spec.tsx

```diff
@@ -12,7 +13,23 @@ import {
 } from "e2e/support/helpers/component-testing-sdk";
 import { getSdkRoot } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
 
-function setup({ locale }: { locale: string }) {
+function setup({
+  locale,
+  instanceLocale,
+}: {
+  locale: string | undefined;
+  instanceLocale?: string;
```

Comment:
nit: use `?` in both fields for consistency

```suggestion
  locale?: string;
  instanceLocale?: string;
```